src/components/ClosedStatus.js

```diff
@@ -73,7 +73,7 @@ const ClosedProposal = ({
 }) => {
   if (approvalFetching)
     return <Loader mr={70} size={20} color="header" background="white" />;
-  if (!eq(proposalAddress, winningProposal.source) || winningProposal === null)
+  if (winningProposal === null || !eq(proposalAddress, winningProposal.source))
```

Comment:
style note: unless the value you're checking has the potential to be an empty string or 0, it's safe to just check `!foo` instead of `foo === null`